src/Ethereum.sol

```diff
@@ -85,7 +86,18 @@ library Ethereum {
     address internal constant WSTETH_DEBT_TOKEN        = 0xd5c3E3B566a42A6110513Ac7670C1a86D76E13E6;
 
     /******************************************************************************************************************/
-    /*** Implementation Addresses                                                                                   ***/
+    /*** SparkLend - Auxiliary Protocol Addresses                                                                   ***/
+    /******************************************************************************************************************/
+
+    address internal constant CAP_AUTOMATOR    = 0x2276f52afba7Cf2525fd0a050DF464AC8532d0ef;
+    address internal constant FREEZER_MOM      = 0xFA36c12Bc307b40c701D65d8FE8F88cCEdE2277a;
+    address internal constant SPELL_FREEZE_ALL = 0xA67d62f75F8D11395eE120CA8390Ab3bF01f0b8A;
+    address internal constant SPELL_FREEZE_DAI = 0x0F9149c4d6018A5999AdA5b592E372845cfeC725;
+    address internal constant SPELL_PAUSE_ALL  = 0x216738c7B1E83cC1A1FFcD3433226B0a3B174484;
+    address internal constant SPELL_PAUSE_DAI  = 0x1B94E2F3818E1D657bE2A62D37560514b52DB17F;
```

Comment:
This should be moved to a separate section for well-known emergency spells.

Reply:
Moved